lib/mongo_client.js

```diff
@@ -124,7 +127,11 @@ function MongoClient(url, options) {
     options: options || {},
     promiseLibrary: null,
     dbCache: {},
-    sessions: []
+    sessions: [],
+    optionsValidationLevel:
```

Comment:
no need to set this here if we go with the "hierarchical" approach outlined below